applications/adv_networking_bench/cpp/testing/test_ano_bench.py

```diff
@@ -241,3 +241,38 @@ def test_gpunetio_single_if_loopback(executable, work_dir, nvidia_nics):
     assert result.returncode == 0, "Application errored out"
     assert "[error]" not in (result.stdout + result.stderr), "Application reported errors"
     assert "[ERR]" not in (result.stdout + result.stderr), "Application reported errors"
+
+
+def test_multi_q_hds_tx_rx(executable, work_dir, nvidia_nics):
```

Comment:
This test passes, but `test_multi_if_loopback` segfaults

Reply:
[4ae941b](https://github.com/nvidia-holoscan/holohub/pull/831/commits/4ae941b8a32be414911b43f0e1a5e1b28f94cdf2)